examples/guestbook/README.md

```diff
@@ -373,66 +408,90 @@ if (isset($_GET['cmd']) === true) {
-frontend
+$ kubectl create -f frontend-service.yaml
+services/frontend
+```
 
+Then, list all your services again:
+
+```shell
 $ kubectl get services
-NAME                    LABELS                                    SELECTOR                     IP                  PORT
-frontend                name=frontend                             name=frontend                10.0.93.211         8000
-redis-master            name=redis-master                         name=redis-master            10.0.246.242        6379
-redis-slave             name=redis-slave                          name=redis-slave             10.0.72.62          6379
+NAME                    LABELS                                    SELECTOR                     IP                  PORT(S)
+frontend                name=frontend                             name=frontend                10.0.93.211         80/TCP
+redis-master            name=redis-master                         name=redis-master            10.0.246.242        6379/TCP
+redis-slave             name=redis-slave                          name=redis-slave             10.0.72.62          6379/TCP
 ```
 
-The pods that we have set up are reachable through the frontend service (10.0.93.211), but you'll notice that this IP is unavailable from outside of the kubernetes cluster. Of course, if you are running kubernetes minions locally, this isn't such a big problem - the port binding will allow you to reach the guestbook website at localhost:80... but the beloved **localhost** solution obviously doesn't work in any real world scenario. If you want the website to be accessible externally, read the *Accessing the guestbook site externally* section below for details.
 
-### Accessing the guestbook site externally
-Kubernetes supports two ways of exposing a service onto an external IP address: `NodePort`s and `LoadBalancer`s , as described [here](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/services.md#external-services). Note that if the `LoadBalancer` way is used, it takes some time for an external IP to show up in `kubectl get services` output.
+#### Accessing the guestbook site externally
+<a href="#accessing-the-guestbook-site-externally"></a>
+
+You'll want to set up your guestbook service so that it can be accessed from outside of the internal Kubernetes network. Above, we introduced one way to do that, using the `type: LoadBalancer` spec.
+
+More generally, Kubernetes supports two ways of exposing a service onto an external IP address: `NodePort`s and `LoadBalancer`s , as described [here](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/services.md#external-services). If the `LoadBalancer` specification is used, it can take some time for an external IP to show up in `kubectl get services` output.
+
+Once you've exposed the service to an external IP, visit the IP to see your guestbook in action. E.g., `http://130.211.188.51:80` in the example above.
+
```

Comment:
This external IP is now shown above. We can paste here another output of "kubectl get services" when "type=LoadBalancer" is specified, the external IP shall be there.